src/commands/protocols/common/protocol-message-command.js

```diff
@@ -25,8 +32,13 @@ class ProtocolMessageCommand extends Command {
     }
 
     async sendProtocolMessage(command, message, messageType) {
-        const { node, operationId } = command.data;
+        const { node, operationId, blockchain } = command.data;
```

Comment:
Did you test if you have blockchain in command data for all protocols?

Reply:
it is missing in get-rpc-controller